spotify_dl/youtube.py

```diff
@@ -1,118 +1,281 @@
 import urllib.request
 from os import path
-
+import os
+import multiprocessing
 import mutagen
-import yt_dlp as youtube_dl
+import csv
+import yt_dlp
 from mutagen.easyid3 import EasyID3
 from mutagen.id3 import APIC, ID3
 from mutagen.mp3 import MP3
-
+from youtube_search import YoutubeSearch
```

Comment:
WHat's this library used for? I don't see it in requirements. In general, I'm not a big fan of libraries that involve scraping YouTube search results because they often break and/or get abandoned, and then the project bears the brunt of it. 

I've already been bit twice before, if you can find a way to not use this, that would be great

Reply:
ohh ok. Have removed it .. didnt know that